qiskit/ignis/measurement/discriminator/iq_discriminators.py

```diff
@@ -496,3 +496,67 @@ def discriminate(self, x_data: List[List[float]]) -> List[str]:
             the discriminated x_data as a list of labels.
         """
         return self._qda.predict(x_data)
+
+
+class SklearnIQDiscriminator(IQDiscriminationFitter):
+    """
+    A generic discriminant analysis discriminator for IQ data that
+    takes an sklearn classifier as an argument.
+    """
+
+    def __init__(self, classifier, cal_results: Union[Result, List[Result]],
+                 qubit_mask: List[int], expected_states: List[str] = None,
+                 standardize: bool = False,
+                 schedules: Union[List[str], List[Schedule]] = None):
+        """
+        Args:
+            classifier:
+                An sklearn classifier to train.
```

Comment:
Perhaps rather:
classifier: A sklearn classifier to train and do the discrimination.

Reply:
Done.